server/src/main/java/org/elasticsearch/action/get/TransportShardMultiGetAction.java

```diff
@@ -174,6 +174,7 @@ private void handleMultiGetOnUnpromotableShard(
         ShardId shardId = indexShard.shardId();
         if (request.refresh()) {
             logger.trace("send refresh action for shard {}", shardId);
+            // TODO: Do we need to pass in shardCountSummary here ?
```

Comment:
I guess we'd look at this while working on get? I think it's a stray todo in this PR though.